index.html

```diff
@@ -645,6 +645,16 @@ <h4>OAuth 2.0</h4>
       </section>
 
     </section>
+    <section>
+      <h3>Options</h3>
+      <p>
+Some of the endpoints defined in the following sections accept an `options` object.
+None of the properties in `options` are required for any endpoint, as these properties
+are intended to meet per-configuration needs that may vary. However, configuration may
+prohibit some options from being used in order to preclude clients from passing certain
+data to a given configuration.
```

Comment:
I think we should also cover the other side:

```suggestion
prohibit some options from being used in order to preclude clients from passing certain
data to a given configuration. Some configurations MAY also require some options to
be passed.
```